website/docs/r/policy_parent_security_policy.html.markdown

```diff
@@ -0,0 +1,160 @@
+---
+subcategory: "Firewall"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_parent_security_policy"
+description: A resource to configure a Security Policy without rules.
+---
+
+# nsxt_policy_parent_security_policy
+
+This resource provides a method for the management of Security Policy without rules.
+
+Note: to avoid unexpected behavior, don't use this resource and resource `nsxt_policy_security_policy` to manage the same Security Policy at the same time.
+To config rules under this resource, please use resource `nsxt_policy_security_policy_rule` to manage rules separately. 
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_parent_security_policy" "policy1" {
+  display_name = "policy1"
+  description  = "Terraform provisioned Security Policy"
+  category     = "Application"
+  locked       = false
+  stateful     = true
+  tcp_strict   = false
+  scope        = [nsxt_policy_group.pets.path]
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "nsxt_policy_security_policy_rule" "rule1" {
+  display_name       = "rule1"
+  description        = "Terraform provisioned Security Policy Rule"
+  policy_path        = nsxt_policy_parent_security_policy.policy1.path
+  sequence_number    = 1
+  destination_groups = [nsxt_policy_group.cats.path, nsxt_policy_group.dogs.path]
+  action             = "DROP"
+  services           = [nsxt_policy_service.icmp.path]
+  logged             = true
+
+  depends_on = [nsxt_policy_parent_security_policy.policy1]
```

Comment:
cosmetic - still some leftover `depends_on` in this doc

Reply:
Done.